examples/vision/mixUp.py

```diff
@@ -0,0 +1,218 @@
+"""
+Title: mixup augmentation for image classification
```

Comment:
MixUp? (capitalized)

Reply:
The authors refer to it as mixup without any capitalization.

Reply:
We still need to capitalize the example title, though.